nexus/src/external_api/http_entrypoints.rs

```diff
@@ -101,16 +101,26 @@ pub fn external_api() -> NexusApiDescription {
         api.register(project_policy_view)?;
         api.register(project_policy_update)?;
 
+        // Customer-Accessible IP Pools API
         api.register(ip_pool_list)?;
         api.register(ip_pool_create)?;
         api.register(ip_pool_view)?;
         api.register(ip_pool_delete)?;
         api.register(ip_pool_update)?;
 
+        // Operator-Accessible IP Pools API
+        api.register(ip_pool_service_view)?;
```

Comment:
FWIW, I tried to ensure that any pools visible from the "ip_pool" endpoints and the "ip_pool_service" endpoints are mutually exclusive - pools returned from one should not be returned from the other.

This is implemented by a few `.filter(rack_id.is_null())` calls in the datastore